fe/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -510,6 +431,14 @@ public void write(DataOutput out) throws IOException {
         out.writeBoolean(disableStreamPreaggregations);
         out.writeInt(parallelExecInstanceNum);
         out.writeInt(exchangeInstanceParallel);
+
+        out.writeLong(loadMemLimit);
+        // false means there are no more variables. It you need to add more variables to persist,
+        // change this to true, and add:
+        //      out.writeXXX(new_var_name);
+        //      out.writeBoolean(false);
+        // By doing this, we no longer need to change the FE meta version when adding new variables to persist.
+        out.writeBoolean(false);   
```

Comment:
I don't this is a elegant way. If we add 100 new variables, the read code will look like ugly.

I think we could serialize and deserialize the whole class by Json.

Reply:
I change it to json.